Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node
+
+WORKDIR /app
```

Comment:
You should move your app to `/opt/first-blog-mern`; follow the [Filesystem Hierarchy Standard](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard) when you can